tests/upgrade/test_existence.py

```diff
@@ -0,0 +1,139 @@
+"""Upgrade TestSuite for validating Satellite entities existence and
```

Comment:
I am not sure if tests should be added to a repository meant to have only system related Fabric tasks, I would suggest having the tests elsewhere.